third_party/2and3/werkzeug/datastructures.pyi

```diff
@@ -56,12 +60,21 @@ class UpdateDictMixin:
     popitem = ...  # type: Any
     update = ...  # type: Any
 
-class TypeConversionDict(dict):
-    def get(self, key, default=None, type=None): ...
-
-class ImmutableTypeConversionDict(ImmutableDictMixin, TypeConversionDict):
-    def copy(self): ...
-    def __copy__(self): ...
+class TypeConversionDict(Dict[_K, _V]):
+    @overload  # type: ignore
+    def get(self, key: _K, default: _V, type: None = ...) -> _V: ...
```

Comment:
Couldn't this be generalized as `default: _T` and then returning `Union[_T, _V]`?